Sources/Misc/SandboxEnvironmentDetector.swift

```diff
@@ -40,12 +44,19 @@ final class BundleSandboxEnvironmentDetector: SandboxEnvironmentDetector {
             return false
```

Comment:
I'd argue if `appStoreReceiptUR` is nil then we are in sandbox environment, so this should be true?

I have not been able to find a case where `appStoreReceiptURL` is nil tho. The URL is always returned, but then the file does not exist in some cases.